Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM alpine:latest
 ENV VERSION 20191124
 WORKDIR /
-RUN apk add build-base cmake curl-dev git --update-cache
+RUN apk add build-base cmake curl-dev ffi-dev git --update-cache
```

Comment:
Weren't we going to use the embedded ffi on Linux and macOS but not on Windows?

Reply:
Yeah probably some leftovers from before I added the embedded ffi.